sites/svelte.dev/src/routes/(authed)/repl/[id]/+page.server.js

```diff
@@ -11,6 +13,6 @@ export async function load({ fetch, params, url }) {
 
 	return {
 		gist,
-		version: url.searchParams.get('version') || '3'
+		version: url.searchParams.get('version') || '3',
```

Comment:
why isn't our Prettier config preventing the trailing comma?

```suggestion
		version: url.searchParams.get('version') || '3'
```

Reply:
That is in the feat/update-content branch. Once that is merged, I'll reformat everything again, then it will be fixed